core/platform/lf_arduino_support.c

```diff
@@ -57,6 +58,10 @@ static volatile int _lf_num_nested_critical_sections = 0;
 static volatile uint32_t _lf_time_us_high = 0;
 static volatile uint32_t _lf_time_us_low_last = 0;
 
+int lf_ack_events() {

```

Comment:
This is only tangentially related to this PR, but I searched this repo and could not find `lf_ack_events` doing anything useful. The only thing I know about it is from [this comment](https://github.com/lf-lang/reactor-c/pull/106#discussion_r962182789), and I'm not sure whether we still need it.

Reply:
My opinion is that we just remove it. We don't need to hide setting/clearing a single flag behind a function

Reply:
The calls to `lf_ack_events()` should be replaced by `_lf_async_event = false`

Reply:
I'm OK with removing it, although I think that hiding the clearing of a flag behind a function is cleaner. I do think that the name problematic (my bad). Maybe `_lf_clear_event_flag` or something like that would be better.